multiqc/utils/search_patterns.yaml

```diff
@@ -82,6 +82,32 @@ biobloomtools:
     num_lines: 2
 bismark/align:
     fn: '*_[SP]E_report.txt'
+biscuit/mapq:
+    - fn: '*_mapq_table.txt'
+    - fn: '*_strand_table.txt'
+    - fn: '*_isize_score_table.txt'
+biscuit/markdup:
+    fn: '*_markdup_report.txt'
```

Comment:
This is quite a generic filename which could conceivably also be used by other tools. It would be good to also add a `content` string to confirm the identity of the file. Otherwise, even if the module code ignores it, the file will not be used by any other modules.

Reply:
I updated the log file format to include a (quite) distinguishable header (BISCUITqc). I updated both search_pattern.yml and the script to detect this header. I hope this reduces the cross-talk between modules. Thanks for pointing me to `content`.